src/main/kotlin/no/nav/familie/ef/sak/iverksett/IverksettingDtoMapper.kt

```diff
@@ -126,7 +124,7 @@ class IverksettingDtoMapper(private val arbeidsfordelingService: Arbeidsfordelin
     }
 }
 
-fun TilkjentYtelse.tilIverksettDto(): TilkjentYtelseDto = TilkjentYtelseDto(
+fun TilkjentYtelse.tilDto(): TilkjentYtelseDto = TilkjentYtelseDto(
```

Comment:
Dobbeltsjekk at disse `tilDto` ikke blir forvirrende/fører til bugs mtp at det finnes en variant for "vår dto" og én for "iverksett sin dto" :)